app/modules/ph/endpoints_ph.py

```diff
@@ -124,21 +127,25 @@ async def create_paper(
         # We only want to send a notification if the paper was released less than a month ago.
         if paper_db.release_date >= now.date() - timedelta(days=30):
             message = Message(
-                context=f"ph-{paper_db.id}",
-                is_visible=True,
                 title=f"📗 PH - {paper_db.name}",
                 content="Un nouveau journal est disponible! 🎉",
-                delivery_datetime=datetime.combine(
-                    paper_db.release_date,
-                    time(hour=8, tzinfo=UTC),
-                ),
-                # The notification will expire in 10 days
-                expire_on=now + timedelta(days=10),
-            )
-            await notification_tool.send_notification_to_topic(
-                custom_topic=CustomTopic(topic=Topic.ph),
-                message=message,
+                action_module="ph",
             )
+            if paper_db.release_date == now.date():
```

Comment:
Could you defer the notification sending even if the delivery date is the current day?

Reply:
Why ?

Reply:
This would simplify the code as the module would not have to check the current date and decide if the notification needs to be planned or send now.
I think this should not have a real impact on performance
 
I believe using the scheduler to send notifications in the future is specific of our Firebase implementation. The module is not forced to know how we implemented this feature to send notifications. For example with the previous implementation the module did not have to check when the notification was needed to be send

Reply:
Not agree